scripts/variantstore/AOU_DELIVERABLES.md

```diff
@@ -1,50 +1,40 @@
+3. `GvsWithdrawSamples` if there are any samples to withdraw from the last callset.
+4. GvsCreateAltAllele
+5. GvsCreateFilterSet (see [naming conventions doc](https://docs.google.com/document/d/1pNtuv7uDoiOFPbwe4zx5sAGH7MyxwKqXkyrpNmBxeow) for guidance on what to name the filter set, which you will need to keep track of for the `GvsExtractCallset` WDL).
+6. GvsPrepareRangesCallset needs to be run twice, once with `control_samples` set to "true" (see [naming conventions doc](https://docs.google.com/document/d/1pNtuv7uDoiOFPbwe4zx5sAGH7MyxwKqXkyrpNmBxeow) for guidance on what to use for `extract_table_prefix` or cohort prefix, which you will need to keep track of for the `GvsExtractCallset` WDL).
+7. GvsExtractCallset needs to be run twice, once with `control_samples` set to "true", and with the `filter_set_name` and `extract_table_prefix` from step 5 & 6.  Include a valid (and secure) "output_gcs_dir" parameter, which is where the VCF and interval list files  will go.
+8. Run [process to create callset statistics](callset_QC/README.md), for which you need
+    1. permission to query table `spec-ops-aou:gvs_public_reference_data.gnomad_v3_sites`
+    2. the data_project you used for all the GVS WDLs
+    3. the default_dataset you used for all the GVS WDLs
+    4. the `extract_table_prefix` input from `GvsExtractCallset` step
+    5. the `filter_set_name` input from `GvsCreateFilterSet` step
+
+## Deliverables (via email once the above steps are complete)
+1. location of the VCF and interval_list files (`output_gcs_dir` input from GvsExtractCallset)
+2. fully qualified name of the BigQuery dataset (input `dataset_name` in the workflows)
 3. [callset statistics](callset_QC/README.md) CSV
 4. [precision and sensitivity results](tieout/AoU_PRECISION_SENSITIVITY.md) TSV
 
-## Cost estimation for BQ queries
-To get cost information for the AoU queries, only the AoU Service account has access.
-After using the json file to auth as that service account, you can query the
-`region-us`.INFORMATION_SCHEMA.JOBS_BY_USER table to narrow down the query you are interested in.
-```
-gcloud auth activate-service-account --key-file=<keyfile>
-# use queries like this to narrow down the time window of the queries you are interested in
-bq --project_id <project> query --use_legacy_sql=false 'select total_bytes_billed, job_id, creation_time, start_time, query from `region-us`.INFORMATION_SCHEMA.JOBS_BY_USER where creation_time > "2021-04-16 00:00:00" and creation_time < "2021-04-18 00:00:00" and total_bytes_billed > 1 order by creation_time asc limit 100'
+## Running the VAT pipeline
+To create a BigQuery table of variant annotations, you may follow the instructions here:
+[process to create variant annotations table](variant_annotations_table/README.md)
+
+The pipeline takes in a jointVCF and outputs a variant annotations table in BigQuery.
```

Comment:
The VAT Readme could certainly use a makeover--but I think it's unwise to have two of them. I will clean up the one referenced here an put up a follow on pr (and then George and Miguel can take a look at it and see it it looks feasible to run)